chapters/revisions.tex

```diff
@@ -1,3070 +1,18 @@
-
-Mattias Klose, Technical University of Berlin, Germany
-
-Nathalie Loubere, Gaz de France, Paris, France
-
-Sven-Erik Mattsson, Dynasim AB, Lund, Sweden
-
-Peter J. Mosterman, DLR Oberpfaffenhofen, Germany
-
-Henrik Nilsson, Linköping University, Sweden
-
-Hans Olsson, , Dynasim AB, Lund, Sweden
-
-Martin Otter, DLR Oberpfaffenhofen, Germany
-
-Per Sahlin, Bris Data AB, Stockholm, Sweden
-
-Andrée Schneider, Fraunhofer Institute for Integrated Circuits, Dresden,
-Germany
-
-Michael Tiller, Ford Motor Company, Dearborn, MI, U.S.A.
-
-Hubertus Tummescheit, Lund Institute of Technology, Sweden
-
-Hans Vangheluwe, University of Gent, Belgium
-
-\subsection{Main Changes in Modelica 1.3}\label{main-changes-in-modelica-1-3}
-
-Modelica 1.3 was released December 15, 1999.
-\begin{itemize}
-\item
-  Defined connection semantics for \lstinline!inner!/\lstinline!outer! connectors.
-\item
-  Defined semantics for protected element.
-\item
-  Defined that least variable variability prefix wins.
-\item
-  Improved semantic definition of array expressions.
-\item
-  Defined scope of \lstinline!for!-loop variables.
-\end{itemize}
-
-\subsection{Main Changes in Modelica 1.2}\label{main-changes-in-modelica-1-2}
-
-Modelica 1.2 was released June 15, 1999.
-\begin{itemize}
-\item
-  Changed the external function interface to give greater flexibility.
-\item
-  Introduced \lstinline!inner!/\lstinline!outer! for dynamic types.
-\item
-  Redefined \lstinline!final! keyword to only restrict further
-  modification.
-\item
-  Restricted redeclaration to replaceable elements.
-\item
-  Defined semantics for \lstinline!if!-clauses.
-\item
-  Defined allowed code optimizations.
-\item
-  Refined the semantics of event-handling.
-\item
-  Introduced \lstinline!fixed! and \lstinline!nominal! attributes.
-\item
-  Introduced \lstinline!terminate! and \lstinline!analysisType!.
-\end{itemize}
-
-\subsection{Main Changes in Modelica 1.1}\label{main-changes-in-modelica-1-1}
-
-Modelica 1.1 was released in December 1998.
-
-Major changes:
-\begin{itemize}
-\item
-  Specification as a separate document from the rationale.
-\item
-  Introduced prefixes \lstinline!discrete! and \lstinline[morekeywords={[2]{nondiscrete}}]!nondiscrete!.
-\item
-  Introduced \lstinline!pre! and \lstinline!when!.
-\item
-  Defined semantics for array expressions.
-\item
-  Introduced built-in functions and operators (only connect was present in Modelica 1.0).
-\end{itemize}
-
-\subsection{Modelica 1.0}\label{modelica-1-0}
+The members of the Modelica Association contributed to the Modelica specification.
 
+% Alternatives:
+% Keep short list all revisions as on specification.modelica.org
+% Only (major revisions, 1.0, 2.0, 3.0)
+% keep nothing
+% or just start.
+% Keeping just the start to have a historic perspective
 Modelica 1, the first version of Modelica, was released in September
 1997, and had the language specification as a short appendix to the
 rationale.
```

Comment:
While we're at it, let's make the whole chapter in its new form have sentence-based line breaks:
```suggestion
Modelica 1, the first version of Modelica, was released in September 1997, and had the language specification as a short appendix to the rationale.
```